### PR TITLE
Bkr241 windows install32bitpuppetagent

### PR DESCRIPTION
**Pull request by kevpl**

### CODE REVIEW

File: lib/beaker/dsl/install_utils/puppet_utils.rb [hunk 1]
```diff
@@ -341,13 +341,17 @@ def install_puppet_from_deb( host, opts )
...
+        # @note on windows, the +:ruby_arch+ host parameter can determine in addition
+        # to other settings whether the 32 or 64bit install is used
         def install_puppet_from_msi( host, opts )
           #only install 64bit builds if
           # - we are on puppet version 3.7+
           # - we do not have install_32 set on host
           # - we do not have install_32 set globally
           version = opts[:version]
-          if !(version_is_less(version, '3.7')) and host.is_x86_64? and not host['install_32'] and not opts['install_32']
+          is_config_32 = true == (host['ruby_arch'] == 'x86') || host['install_32'] || opts['install_32']
```
Comment (joshcooper) [inline lib/beaker/dsl/install_utils/puppet_utils.rb]: Seems like `true == ...` comparison can be dropped.

File: lib/beaker/dsl/install_utils/puppet_utils.rb [hunk 2]
```diff
@@ -721,7 +729,12 @@ def install_puppetagent_dev_repo( host, opts )
           when /^windows$/
             release_path << 'windows'
             onhost_copy_base = '`cygpath -smF 35`/'
-            arch_suffix = arch =~ /64/ ? '64' : '86'
+            is_config_32 = true == (host['ruby_arch'] == 'x86') || host['install_32'] || opts['install_32']
```
Comment (joshcooper) [inline lib/beaker/dsl/install_utils/puppet_utils.rb]: Same comment here about `true == ...`

**Comment by anodelman** [comment]
retest this please

**Comment by anodelman** [comment]
Failure unrelated to patch.

**Pull request merged**